### DISCUSSION TITLE
Increase EON-information

### DISCUSSION THREAD

**Discussion by lxsxl** [PE5PVB/TEF6686_ESP32#337] [Ideas]
A few days we addressed the idea of increasing EON-information (TP, TA, PTY for the other networks). Is this a subject for the actual developement? Or better for a future release? Or not possible at all due to the necessarity of dividing the EON/AF-screen (as a side-effect the AF-screen could display more AF at once)?

**Comment by PE5PVB** [comment]
Everything is possible, but is it usefull to know that information?

**Comment by lxsxl** [comment]
Many things you created for the TEF can be discussed if it's useful or not. It's more like "now that we've come so far...". I would say, those EON-contents are the last subjects that are not displayed yet (if we leave out TMC). So it would be great to fill this last gap.

**Comment by lxsxl** [comment]
As this is handled in https://github.com/PE5PVB/TEF6686_ESP32/discussions/393 (and almost finished) I close this discussion.
